core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -1121,6 +1120,9 @@ def __create_boot_dot_properties(self):
 
         for server in servers:
             properties = Properties()
+
+            admin_username = self.aliases.decrypt_password(admin_username)
```

Comment:
Why do you need to decrypt it each time there is a server (for every server)?  Shouldn't this decrypt step be done near line 1110 (outside of the for-loop)?

Reply:
Yea, I know about that.  Both decrypt and encrypt of credentials can be outside the loop, I also change to create boot.properties when it is not in production mode